dftimewolf/lib/validators/manager.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""Manager class for validators."""
+from typing import Dict, List, Optional, Sequence, Type
+
+from dftimewolf.lib import errors, resources, args_validator
+
+
+class ValidatorsManager:
+  """Class that handles validating arguments."""
+
+  _validator_classes = {}  # type: Dict[str, Type['args_validator.AbstractValidator']] # pylint: disable=line-too-long
+
+  @classmethod
+  def ListValidators(cls) -> List[str]:
+    """Returns a list of all registered validators.
+
+    Returns:
+      A list of all registered validators.
+    """
+    return list(cls._validator_classes.keys())
+
+
+  @classmethod
+  def RegisterValidator(
```

Comment:
Other manages have a boolean parameter that allows for overwriting validators (useful if we have different internal validation logic)

Reply:
Added an override argument here